pout/compat.py

```diff
@@ -12,7 +12,7 @@
     unicode = unicode
     range = xrange # range is now always an iterator
 
-    from collections import Callable, Iterable, Set
+    from collections.abc import Callable, Iterable, Set
```

Comment:
This line is gated with an `is_py2` check, and in Python 2 the [correct way to import the ABC classses is from the base collections module](https://docs.python.org/2/library/collections.html#collections-abstract-base-classes). You'll notice that the [is_py3 importing](https://github.com/Jaymon/pout/blob/db34e507c05e565a4ed665b985195f14e9074e7a/pout/compat.py#L53) uses the `collecitons.abc` module

Reply:
oops. gonna fix.